src/Illuminate/Broadcasting/BroadcastManager.php

```diff
@@ -124,7 +123,7 @@ public function queue($event)
             $queue = $event->queue;
         }
 
-        $this->app->make('queue')->connection($connection)->pushOn(
+        $this->app->make('queue')->connection($event->connection ?? null)->pushOn(
```

Comment:
Can I ask why `$event->connection ?? null` is used?
I may misunderstand the `??` operator, but as I understand it it checks if the connection is null and if it is, don't take it but the second "argument", which is a hard written `null` here.

Reply:
In the event that $event doesn't have a connection property, the `??` suppresses the error:

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/2829600/70868032-15c32780-1f74-11ea-8edf-176ead9afdf9.png">

Reply:
Thanks, didn't know it also checks `$event` itself.